modules/migration-mtc-cr-manifests.adoc

```diff
@@ -28,7 +28,7 @@ spec:
   destMigClusterRef:
     name: <destination_cluster_ref>
     namespace: openshift-migration
-  namespaces:
+  namespaces: <1>
```

Comment:
@apinnick Can you confirm if these callouts (here and in the below file) have been added at the correct location?

Reply:
@bergerhoffer 
Yes, this is correct.